ElixirLS.novaextension/extension.json

```diff
@@ -43,6 +43,12 @@
       "description": "Automatically format a file on save",
       "type": "boolean",
       "default": false
+    },
+    {
+      "key": "elixir-ls.projectDir",
+      "title": "Project Directory",
+      "description": "Add configuration option for when your Mix project is in a subdirectory instead of the workspace root.",
+      "type": "string"
```

Comment:
Can you add a default here? I believe the default is "./" but we'll need to test that works for projects with the mix file in the root.

Reply:
done!